client/auth/style.scss

```diff
@@ -57,7 +57,7 @@ html.is-desktop {
 
 		.form-fieldset input {
 		position: relative;
-		z-index: 1;
+		z-index: z-index( 'root', 'auth__form__input' );
```

Comment:
Can you explain what the naming convention is here for `auth__form__input` ? Is the idea to use double-underscore in place of a space in the selector? Should we prescribe this in the `css.md` docs?

Also, is there any reason we couldn't just use the full selector, spaces and all, in the string?

```sass
z-index: z-index( 'root', '.auth__form .form-fieldset input' );
```